docs/source/installation/2-manual-install.rst

```diff
@@ -589,6 +589,12 @@ initially.
 
        $ commcare-cloud cluster deploy
 
+   Or if you need to deploy a specific version of CommCare HQ as opposed to the latest:
+
+   ::
+
+       $ commcare-cloud cluster deploy --commcare-rev=<commit-hash>
```

Comment:
not blocking but might be good to clarify the specific scenarios in which you'll need to deploy with a specific version, unless they read this already knowing they have to (like if they were redirected from the project migration docs or something)

Reply:
Yeah the docs for a data migration mention that they will need to deploy from a specific commit, and we will share a commit hash with them which should also remind them that they will need to use it. I'd rather not add too much to the manual install docs about this, but I imagine there is still room for improvement.